src/main/scala/com/microsoft/hyperspace/actions/RefreshActionBase.scala

```diff
@@ -101,60 +101,49 @@ private[actions] abstract class RefreshActionBase(
    * are the files for which the index was never updated in the past.
    */
   protected lazy val deletedFiles: Seq[String] = {
-    val rels = previousIndexLogEntry.relations
-    val originalFiles = rels.head.data.properties.content.fileInfos
-    val currentFiles = rels.head.rootPaths
-      .flatMap { p =>
-        Content
-          .fromDirectory(path = new Path(p), throwIfNotExists = true)
```

Comment:
@imback82 BTW this change is required for this release because these lines look up filesystem directly, not from the refresh df. (though the index log entry & source.Content are generated based on the refresh df)

Reply:
OK, let me check

Reply:
The refreshed `df` also does the filessystem look up. And since both are doing it on the same `rootPaths`, wouldn't the result be the same (for non-partitioned data)?

Reply:
Yes but a file can be deleted between 2 look up point.

Reply:
Got it. Could you update the PR description with this bug fix as well for the record? Thanks!